app/src/main/java/mozilla/lockbox/presenter/ItemListPresenter.kt

```diff
@@ -49,7 +53,21 @@ class ItemListPresenter(
                 }
                 .addTo(compositeDisposable)
 
+        view.menuItemSelections
+            .subscribe(this::onMenuItem)
+            .addTo(compositeDisposable)
+
         // TODO: remove this when we have proper locking / unlocking
         dispatcher.dispatch(DataStoreAction.Unlock)
     }
+
+    fun onMenuItem(@IdRes item: Int) {
+        val action = when (item) {
+            R.id.action_itemList_to_locked, R.id.fragment_locked -> RouteAction.LockScreen
```

Comment:
in what situation would the `menuItemSelection` return an `R.id.action_...`? unfamiliar with exactly how the `nav_menu` fits in here (more a personal question than review issue)

Reply:
Currently, there is no merit in having this accept both the action and the destination. 

I'm hard pressed to justify this, except the vague feeling we should move the the action to be a global action. On the whole, that should be invisible to this code.